### DISCUSSION TITLE
[语雀] 知识网络功能

### DISCUSSION THREAD

**Discussion by tyn1998** [X-lab2017/open-wonderland#113] [1️⃣ 新鲜事]
似乎是新功能~

<img width="1191" alt="image" src="https://user-images.githubusercontent.com/32434520/176651288-d4abe8d9-c9e7-4430-afa8-1970ad234f22.png">

**Comment by will-ww** [comment]
前面也注意到了，这个具体有啥用？

**Reply by bifenglin** [reply]
昨天会议我还打算提一下这个，这个基本类似于论文引用关系，添加引用关系的。

**Reply by will-ww** [reply]
嗯，是一个~
